src/actions/hover.rs

```diff
@@ -878,7 +885,7 @@ fn format_method(rustfmt: Rustfmt, fmt_config: &FmtConfig, the_type: String) ->
 pub fn tooltip(
     ctx: &InitActionContext,
     params: &TextDocumentPositionParams,
-) -> Result<Vec<MarkedString>, ResponseError> {
+    ) -> Result<Tooltip, ResponseError> {
```

Comment:
```suggestion
) -> Result<Tooltip, ResponseError> {
```
I believe that no leading whitespace is preferred by rustfmt and used elsewhere here